hydromt_fiat/fiat.py

```diff
@@ -634,6 +646,25 @@ def setup_hazard(
                     if self.exposure.unit != da.units:
                         da = da * unit_conversion_factor
 
+                                # clip bonding boxes and save new exposure
```

Comment:
I am not sure if I am really fond of this part. Right now this could lead in huge changes in the exposure model if I decide to provide a hazard map that has a really small extent. I think there should be at least a warning or some info logged on this change. 
Moreover, I think the location of the exposure object being inside the hazard bound should be already taken care of when creating the exposure vector. So the bounds of the hazard should be the "region" that is used when we build the model. If this is used but still there are objects that fall outside the hazard bounds, it would mean that something is going wrong there, right?